Content.Server/Mining/MeteorComponent.cs

```diff
@@ -0,0 +1,22 @@
+using Content.Shared.Damage;
+
+namespace Content.Server.Mining;
+
+/// <summary>
+/// This is used for meteors which hit objects, dealing damage to destroy/kill the object and dealing equal damage back to itself.
+/// </summary>
+[RegisterComponent]
+public sealed partial class MeteorComponent : Component
+{
+    /// <summary>
+    /// percentage distribution of damage applied. Is multiplied by the actual damage it needs to deal.
```

Comment:
Nothing in code or YAML is enforcing that all of the values are less than or equal to one, or that all of the damage sums to one. I can imagine someone accidentally writing blunt 10, slash 5 and accidentally making a mega-meteor.

Perhaps it would be better to always normalize `DamageDistribution` when this gets used.

"Distribution" is also kind of a misnomer, because I associate that with a probability distribution. Perhaps just `DamageTypes` would be enough.

Reply:
i think it's fine for people to increase the value to above 1. It's not super practical but it could be used to create an "overkill" meteor that gibs players and burns through things faster. In any case, this is really only set once on the base meteor and isn't changed for variants so it's not really going to be a common thing people fiddle with.